templates/agent/src/app/api/agent/route.ts

```diff
@@ -1,9 +1,10 @@
 import {basicAgent} from '~/agents/basic'
 
 export async function POST(req) {
-	const { input } = await req.json()
+	const { input, botName } = await req.json()
+	console.log("Using OpenAI", botName)
```

Comment:
Remove console.log

Reply:
It might be useful in development so the dev can see exactly which model they are using as this is not shown by the langchain call and is good to know since accidentally using gpt-4 can be pricey. I can make it so that the console.log is only shown when running in dev (ex: bun run dev). What do you think?

Reply:
Made it only log in dev mode in the new commits